packages/data/src/world/oot/jabu_jabu.yml

```diff
@@ -86,12 +86,12 @@
   dungeon: JJ
   exits:
     "Jabu-Jabu Basement": "true"
-    "Jabu-Jabu Elevator": "has_ranged_weapon || has_bombs" #|| Has soul_ruto && settingRutoF1=false
+    "Jabu-Jabu Elevator": "(can_swim || is_adult || time_travel_at_will || has_hover_boots) && (has_ranged_weapon || has_bombs)" #|| Has soul_ruto && settingRutoF1=false
```

Comment:
time travel is not needed here, since it already can detect you can swap ages. time travel is used in other logic if you need to get a location specifically by age swapping back and forth, such as the Kak Ladder GS (age swap, climb up as adult, age swap, use din's)